### PR TITLE
Add Flashlist plugin

### PR DESCRIPTION
**Pull request by pooyahrtn**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -22,9 +22,7 @@
       "react-native@0.48.3",
       "react-native-electrode-bridge@1.5.10"
     ],
-    "targetJsDependencies": [
-      "react@16.0.0-alpha.12"
-    ]
+    "targetJsDependencies": ["react@16.0.0-alpha.12"]
```
Comment (pooyahrtn) [inline manifest.json:25]: Sorry for the mess, don't know how to disable prettier in this file. The only change is `flash-list` being added on lin 1450.

File: plugins/ern_v0.48.0+/@shopify/flash-list_v1.0.4+/config.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "android": {
+    "root": "",
+    "moduleName": "android",
+    "modulePath": "src/main/kotlin",
+    "dependencies": [
+      "androidx.appcompat:appcompat:1.2.0",
+      "androidx.core:core-ktx:1.6.0"
+    ],
+    "copy": [{ "source": "android/", "dest": "lib/src/main/assets/fonts" }]
```
Comment (FlorBosch) [inline plugins/ern_v0.48.0+/@shopify/flash-list_v1.0.4+/config.json]: Shouldn't the `source` be more specific? or the only things inside the android folder are the fonts? 🤔

Comment (pooyahrtn) [reply plugins/ern_v0.48.0+/@shopify/flash-list_v1.0.4+/config.json]: The weirder thing is that this package doesn't even have any fonts :D I simply copied the configuration from `gesture-handler`. I guess `ern` just needs to have a plugin defined in this case.

Comment (FlorBosch) [reply plugins/ern_v0.48.0+/@shopify/flash-list_v1.0.4+/config.json]: ohh then can't the "copy" property be removed?

Comment (pooyahrtn) [reply plugins/ern_v0.48.0+/@shopify/flash-list_v1.0.4+/config.json]: It was actually not necessary. I removed it [a71b367](https://github.com/PicnicSupermarket/electrode-native-manifest/pull/11/commits/a71b367ead27cdecb45a566482f7db28358a17f5) here.

**Review by FlorBosch — APPROVED** [review]

**Review by LcTwisk — APPROVED** [review]

**Pull request merged**